### DISCUSSION TITLE
Can I use RHCOS for OKD ?

### DISCUSSION THREAD

**Discussion by MaroWielki** [okd-project/okd#658] [Q&A]
Hi,

Can I run the OKD cluster on RedHat CoreOS instead of Fedora CoreOs?

If so, what would be the procedure to setup such a cluster? Should I just replace the FCOS images with RHCOS during the installation or will it require more effort?

**Comment by vrutkovs** [comment]
It should be possible, yes. Note that you should create your own payload with `machine-os-content` image replaced from OCP payload, so that CVO would not revert your installation back to Fedora CoreOS
